packages/hoppscotch-app/components/lenses/renderers/composables/useDownloadResponse.ts

```diff
@@ -0,0 +1,35 @@
+import { Ref, ref } from "@nuxtjs/composition-api"
+
+export type downloadResponseReturnType = (() => void) | Ref<any>
+
+export default function useDownloadResponse(
+  contentType: string,
+  responseBodyText: Ref<any>,
```

Comment:
The type is broad. There are only two possibilities for the type of Response Body Ref. In this case its a string. Narrow the type down please.